packages/blockchain-wallet-v4-frontend/src/components/Send/LockTime/index.tsx

```diff
@@ -24,7 +24,7 @@ class LockTimeContainer extends React.PureComponent<Props> {
         <LockTime
           {...rest}
           lockTime={moment
-            .duration(val.withdrawLockCheck.lockTime, 'seconds')
+            .duration(val.withdrawLockCheck?.lockTime, 'seconds')
```

Comment:
Is there a default lockTime amount in case `withdrawLockCheck` is undefined?

Reply:
I guess this is not typed correctly